Orange/regression/linear.py

```diff
@@ -82,13 +83,33 @@ def __init__(self, loss='squared_loss', alpha=0.0001, epsilon=0.1,
         self.params = vars()
 
     def fit(self, X, Y, W):
-        sk = skl_linear_model.SGDRegressor(**self.params)
+        sk = self.__wraps__(**self.params)
         clf = skl_pipeline.Pipeline(
             [('scaler', skl_preprocessing.StandardScaler()), ('sgd', sk)])
         clf.fit(X, Y.ravel())
         return LinearModel(clf)
 
 
+class PolynomialLearner(SklLearner):
+    __wraps__ = None
+    name = 'unireg'
```

Comment:
Why is the name 'unireg'?
This is a general polynomial learner that can work with multiple variables.
Maybe 'poly learner' or even 'poly ' + learner.name?